tests/data/drivers/test_sumologic_driver.py

```diff
@@ -0,0 +1,219 @@
+# -------------------------------------------------------------------------
```

Comment:
See the branch that I submitted. I couldn't get any of these tests to work. Seems like it was based on the Splunk driver test code and has a lot of unrelated stuff from that.